packages/syft/src/syft/core/node/common/action/run_class_method_action.py

```diff
@@ -243,7 +243,8 @@ def execute_action(self, node: AbstractNode, verify_key: VerifyKey) -> None:
             args=tag_args,
             kwargs=tag_kwargs,
         )
-
+        # TODO :Make it work for inplace ops in sql alchemy.
+        node.store[self._self.id_at_location] = resolved_self  # type: ignore
```

Comment:
Q: This seems wrong (that we have 2 `id_at_location`) one in `self` and one in `self._self`?

Reply:
Great question george, self._self points to ``current pointer being manipulated``, self.id_at_location points to the result, while doing inplace operations , when we manipulate store object of that pointer , the current pointer is not automatically mutated in sql alchemy.

Reply:
So we reassign back to the store the  ``Storeableobject`` after manipulation.

Reply:
Yes this is correct and the tests verify so.